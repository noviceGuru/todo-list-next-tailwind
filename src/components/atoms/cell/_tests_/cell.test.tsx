import { render, screen } from '@testing-library/react'

import Cell from '../cell'

it('renders cell with the correct text and looks', () => {
	render(
		<Cell
		text='cell fake text'
		/>)

	const cell = screen.getByText("cell fake text")
	
	expect(cell.textContent).toBe('cell fake text')
	expect(cell).toMatchSnapshot()
})