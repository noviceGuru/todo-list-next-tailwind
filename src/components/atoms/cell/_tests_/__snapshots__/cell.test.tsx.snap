// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders cell with the correct text and looks 1`] = `
<span>
  cell fake text
</span>
`;
